README.md

```diff
@@ -696,6 +706,33 @@ public String startHelloWorldProcess() {
 }
 ```
 
+### Camunda Platform Assertions - Multiple process tests
+If you want to create multiple process tests, you need to implement the following initialisation code in each test. 
+This makes the assertions aware of your process engine. Otherwise, it tries to reuse the engine of the test that got 
+executed first and that may already be shut down. See [Camunda Platform Assert User Guide](https://github.com/camunda/camunda-bpm-assert/blob/master/docs/README.md#using-a-non-default-process-engine).
+
+```java
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.camunda.bpm.engine.ProcessEngine;
+import org.junit.jupiter.api.BeforeEach;
+
+import javax.inject.Inject;
+
+import static org.camunda.bpm.engine.test.assertions.bpmn.BpmnAwareTests.*;
```

Comment:
Hi @martisaw ,

please write 
```
import static org.camunda.bpm.engine.test.assertions.bpmn.BpmnAwareTests.init;
```

then everybody should understand what happens in setUp Method.

Reply:
We decided to keept the BpmnAwareTests.* because if you implement a process test you will need this import.